Sources/UIView.swift

```diff
@@ -140,7 +140,10 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
 
     public init(frame: CGRect) {
         self.layer = type(of: self).layerClass.init()
-        self.layer.contentsScale = UIScreen.main.scale
+        if let uiScreenMain = UIScreen.main {
```

Comment:
`UIScreen.main` is an implicitly unwrapped optional: https://github.com/flowkey/UIKit-cross-platform/blob/196ab113ab457a2c393689317b9fe6c067c75fd7/Sources/UIScreen.swift#L16